rotkehlchen/db/filtering.py

```diff
@@ -1181,13 +1191,14 @@ def make(  # type: ignore  # it is expected to be incompatible with supertype
             ignored_ids: Optional[list[str]] = None,
             null_columns: Optional[list[str]] = None,
             event_identifiers: Optional[list[str]] = None,
-            entry_types: Optional[list[HistoryBaseEntryType]] = None,
+            entry_types: Optional[Dict[str, Union[list[HistoryBaseEntryType], str]]] = None,
             exclude_ignored_assets: bool = False,
             tx_hashes: Optional[list[EVMTxHash]] = None,
             validator_indices: Optional[list[int]] = None,
     ) -> 'EthDepositEventFilterQuery':
         if entry_types is None:
-            entry_types = [HistoryBaseEntryType.ETH_DEPOSIT_EVENT]
+            entry_type_values = [HistoryBaseEntryType.ETH_DEPOSIT_EVENT]
+            entry_types = IncludeExcludeListField.from_values(entry_type_values)
```

Comment:
I am not sure if the method `from_values` is worth it. I think it will be better to construct a tuple / dict / whatever structure you will choose for the filter right here. In order to explicitly specify here that the values should be included.

Reply:
I was thinking of having such a function so that we can create these objects easily from a list instead of creating a dictionary everytime you want to use it. But maybe it doesn't worth it and it adds complexity insread. I removed this helper function.